Gems/EMotionFX/Code/MCore/Source/StringConversions.cpp

```diff
@@ -73,64 +73,3 @@ namespace MCore
         return result;
     }
 }
-
-namespace AZStd
-{
-    void to_string(string& str, bool value)
-    {
-        str = value ? "true" : "false";
-    }
-
-    void to_string(string& str, const AZ::Vector2& value)
-    {
-        str = AZStd::string::format("%.8f,%.8f", 
```

Comment:
Not a big deal and probably a matter of personal preference, but a space after the commas might make things easier for readers to parse.